.travis.yml

```diff
@@ -3,12 +3,13 @@ cache:
   directories:
     - WcaOnRails/vendor/bundle
 rvm:
-  - 2.3.1
+  - 2.4.0
```

Comment:
Travis seems to also ignore the `.ruby-version` file. See [here](https://travis-ci.org/thewca/worldcubeassociation.org/builds/209357292#L402).

For now, I've gone back to just specifying the ruby version in `.travis.yml`.